Synopse.inc

```diff
@@ -242,11 +242,13 @@
     {$define ISFPC32}
   {$endif}
   {$ifdef VER3_2}
+    {$define FPC_PROVIDE_ATTR_TABLE}

```

Comment:
this is not correct, and would break compilation on 3.2/3.3

Reply:
have you tried it ?
i have it with 3.2 in use and the actual implementation dosend work...
can we please discuss this before you close the pull requests ?
or do you dont want that somebody help you ?